### DISCUSSION TITLE
How to check table schema in Supabase to check foreign key of a table column

### DISCUSSION THREAD

**Discussion by StevePhuc** [supabase/supabase#2973] [Questions] [answered]
I want to check foreign key of a table column. how to do it in Supabase.

**Comment by soedirgo** [comment, marked as answer]
Hey @StevePhuc, if you view the table in the Table Editor and edit the column, you can see the foreign key for that column.

https://user-images.githubusercontent.com/31685197/131211320-43443349-cf6d-4e53-aca6-435113a5213a.mov
